shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -222,12 +225,10 @@ public String getUsage() {
 
   @Override
   public String getDescription() {
-    return "Displays information for all files and directories directly under the specified path."
-        + " Specify -d to list directories as plain files."
```

Comment:
Are the flags defined elsewhere?

Reply:
yes. the description is already in the def of each option.  So `help ls` will see duplicated description on the flags.